apps/ledger-live-mobile/e2e/specs/deeplinks.spec.ts

```diff
@@ -27,25 +27,29 @@ let ethereumLong: string = "Ethereum";
 let bitcoinLong: string = "Bitcoin";
 
 let deviceName = "Nano X de David";
-let mercuryoDL: string = "Mercuryo";
+let mercuryoDL = { name: "Mercuryo", url: " https://www.mercuryo.io/" };
```

Comment:
I think these can all be `const` and don't require the `:string` types as it's inferred from the strings themselves